lfx-mentorship/2023/01-Mar-May/project_ideas.md

```diff
@@ -45,3 +45,9 @@ Upstream Issue: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/issu
 - Recommended Skills: golang
 - Mentor: Rohit Nayak (@rohit-nayak-ps, rohit@planetscale.com) 
 - Upstream Issue: https://github.com/vitessio/vitess/issues/12136
+
+##### Add complete parsing support for Spatial MySQL functions
+- Description: Vitess is a database clustering system for horizontal scaling of MySQL. One of the key goals of Vitess is to emulate MySQL behavior even while running multiple MySQL instances so that ORMs and frameworks work seamlessly. Vitess has its own in-built SQL-parser which it uses to understand the query and represent as structs for further processing. As of now, a lot of spatial MySQL functions are not parsed correctly and result in syntax errors. The task of the mentee would be to add parsing support for such functions and features which can be found at https://dev.mysql.com/doc/refman/8.0/en/spatial-analysis-functions.html
+- Recommended Skills: go, SQL, yacc, compilers and lexers
+- Mentor(s): [Manan Gupta](https://github.com/GuptaManan100)
```

Comment:
Could you add your email address as well @GuptaManan100?

Reply:
Added my email.